### PR TITLE
MA-410 Add course_id on import for existing VAL entry.

### PR DESCRIPTION
**Pull request by nasthagiri**
https://openedx.atlassian.net/browse/MA-410

### CODE REVIEW

**Comment by nasthagiri** [comment]
@gwprice @BenjiLee Please review.

File: edxval/api.py [hunk 1]
```diff
@@ -428,21 +428,30 @@ def import_from_xml(xml, edx_video_id, course_id=None):
...
+        video = _get_video(edx_video_id)
         logger.info(
             "edx_video_id '%s' present in course '%s' not imported because it exists in VAL.",
             edx_video_id,
             course_id,
         )
+        if course_id:
+            CourseVideo.objects.get_or_create(
+                video=video,
+                course_id=course_id
```
Comment (gwprice) [inline edxval/api.py]: course_id needs to be validated

File: edxval/api.py [hunk 2]
```diff
@@ -428,21 +428,30 @@ def import_from_xml(xml, edx_video_id, course_id=None):
     if xml.tag != 'video_asset':
         raise ValCannotCreateError('Invalid XML')
 
-    if Video.objects.filter(edx_video_id=edx_video_id).exists():
+    try:
+        video = _get_video(edx_video_id)
```
Comment (gwprice) [inline edxval/api.py]: _get_video does unnecessary prefetching. It might be best to add a prefetch argument to _get_video.

**Comment by BenjiLee** [comment]
:+1: when Greg's comments are addressed.

**Comment by nasthagiri** [comment]
@gwprice Please re-review.

**Comment by gwprice** [comment]
:+1:

**Comment by gwprice** [comment]
I just realized that there should be another test case for this change (one that I think will currently fail). A video that is already associated with a specific course should be re-imported for that course.

**Pull request merged**